### ISSUE TITLE
Sprite glitch with the T-18

### ISSUE THREAD

**Issue by jdeuce22** [tgstation/TerraGov-Marine-Corps#5908]
## Testmerges:
5887: 'Maptick experiment' by TiviPlus at commit f57c7b638d
5860: 'Add new supply drop console' by BraveMole at commit 845cc5e587
5811: 'Jetpack' by BraveMole at commit 45086db45c
5803: 'Xenos can chose a xenomorph or a silo to track' by BraveMole at commit 548d409e59
5695: 'Neurospit Tweak: Stamina Damage Nerfed, Weak Neurogas Spawn and Neurotoxin Reagent Transfer Added' by Surrealaser at commit 4738437acb
5662: 'Revert "auxtools debugger (#5530)"' by TiviPlus at commit e64a73edb2
(I think that's what you meant?)

## Reproduction:
basically, the issue is that when you place certain attachments onto the T-18 they get placed "floating" off of the sprite, the attachments I used were the mag harness and BC
